coremltools/models/neural_network/builder.py

```diff
@@ -13,7 +13,7 @@
 from .._interface_management import set_transform_interface_params
 from .. import datatypes
 import numpy as np
-
+from quantization_utils import unpack_to_bytes, _convert_array_to_nbit_quantized_bytes
```

Comment:
This seems problematic for python 3. I think it should be `from .quantization_utils`